package.json

```diff
@@ -11,7 +11,8 @@
     "storybook": "NODE_ENV=development start-storybook -p 8080 -c .storybook",
     "storybook:export": "build-storybook -c .storybook -o public",
     "speculate": "speculate",
-    "test": "jest --verbose"
+    "test": "jest --verbose",
```

Comment:
Shall we copy [Simorgh's approach](https://github.com/BBC-News/simorgh/blob/latest/package.json#L23) and have this run `npm run test:lint && npm run test:unit`?